src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -2393,6 +2417,22 @@ def check_raw_parameters(self):
         )
         raise RequiredArgumentMissingError(error_msg)
 
+    def update_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
+        """Update outbound type of network profile for the ManagedCluster object.
```

Comment:
It seems to me that most of the properties of the network profile could not be updated, not sure if we could find a name with smaller scope just like `update_outbound_type_in_network_profile`.

Also please correct the comment of this func, not updating outbound type here

Reply:
changed to "update_network_plugin_settings"